Lib/LibStatic/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ IF(WIN32 AND NOT UNIX)
 		IF( MSVC10 OR MSVC11 OR MSVC12 )
 			set_target_properties(RakNetLibStatic PROPERTIES STATIC_LIBRARY_FLAGS "/NODEFAULTLIB:\"LIBCD.lib LIBCMTD.lib MSVCRT.lib\"" )
 		ELSE()
-			set_target_properties(RakNetLibStatic PROPERTIES STATIC_LIBRARY_FLAGS "/NODEFAULTLIB:&quot;LIBCD.lib LIBCMTD.lib MSVCRT.lib&quot;" )
+			set_target_properties(RakNetLibStatic PROPERTIES STATIC_LIBRARY_FLAGS "/NODEFAULTLIB:\"LIBCD.lib LIBCMTD.lib MSVCRT.lib\"" )

```

Comment:
This looks kind of flawed. Please note that the section you modified was used to generate VS2005/2008 project files for which, as far as my understanding goes, the &quot;-usage was required.

The proper "fix" to ensure that VS2015/VS2017 generated project files use the correct settings would be to either add MSVC14 to the list in ln 20 (see #64 for example) and/or drop the entire if/else clause here and solely go with the \"-style (i.e. drop support for VS 2005/2008 which we did in SLikeNet now - the change will be available shortly in the SLikeNet Git and SVN repositories - https://github.com/SLikeSoft/SLikeNet ).